### ISSUE TITLE
DownloadedMovieImportService Unknown Movie for file

### ISSUE THREAD

**Issue by shaggibruv** [Radarr/Radarr#742]
I seem to be having a few problems with Radarr picking up completed downloads and matching them to the correct movie, one example i've tried renaming a few times then putting it in the drone folder to scan.

Radarr Version: 0.2.0.344
Mono Version: 4.2.1 (Debian 4.2.1.102+dfsg2-7ubuntu4)

Logs:
17-2-14 00:34:09.8|Debug|Parser|Parsing string 'Adulthood 2008 720p BluRay Dd5.1 x264-RIGHTSIZE'
17-2-14 00:34:09.8|Debug|Parser|Movie Parsed. Adulthood - 0 
17-2-14 00:34:09.8|Debug|Parser|Language parsed: English
17-2-14 00:34:09.8|Debug|QualityParser|Trying to parse quality for Adulthood 2008 720p BluRay Dd5.1 x264-RIGHTSIZE
17-2-14 00:34:09.8|Debug|Parser|Quality parsed: Bluray-720p v1
17-2-14 00:34:09.8|Debug|Parser|Release Group parsed: RIGHTSIZE
17-2-14 00:34:10.5|Debug|DownloadedMovieImportService|Unknown Movie Adulthood 2008 720p BluRay Dd5.1 x264-RIGHTSIZE
17-2-14 00:34:10.6|Debug|Api|[GET] /api/system/task??sort_by=name&order=asc: 200.OK (24 ms)
17-2-14 00:34:10.8|Debug|Api|[GET] /api/system/task??sort_by=name&order=asc: 200.OK (49 ms)
17-2-14 00:34:12.8|Debug|Api|[GET] /api/log/file: 200.OK (7 ms)
17-2-14 00:34:12.9|Debug|Api|[GET] /api/log/file/radarr.debug.txt: 200.OK (8 ms)

17-2-14 00:35:38.8|Debug|Parser|Parsing string 'Adulthood'
17-2-14 00:35:38.8|Debug|Parser|Unable to parse Adulthood
17-2-14 00:35:39.2|Debug|DownloadedMovieImportService|Unknown Movie Adulthood
17-2-14 00:35:39.2|Debug|DiskScanService|Scanning '/mnt/ram/data_out/Radarr_drone/' for video files
17-2-14 00:35:39.2|Debug|DiskScanService|1 video files were found in /mnt/ram/data_out/Radarr_drone/
17-2-14 00:35:39.2|Debug|Parser|Parsing string 'Adulthood 2008 720p BluRay DD5.1 x264-RightSiZE'
17-2-14 00:35:39.2|Debug|Parser|Movie Parsed. Adulthood - 0 
17-2-14 00:35:39.2|Debug|Parser|Language parsed: English
17-2-14 00:35:39.2|Debug|QualityParser|Trying to parse quality for Adulthood 2008 720p BluRay DD5.1 x264-RightSiZE
17-2-14 00:35:39.2|Debug|Parser|Quality parsed: Bluray-720p v1
17-2-14 00:35:39.2|Debug|Parser|Release Group parsed: RightSiZE
17-2-14 00:35:40.2|Debug|DownloadedMovieImportService|Unknown Movie for file: Adulthood 2008 720p BluRay DD5.1 x264-RightSiZE.mkv

17-2-14 00:39:05.3|Debug|Parser|Parsing string 'Adulthood'
17-2-14 00:39:05.4|Debug|Parser|Unable to parse Adulthood
17-2-14 00:39:05.7|Debug|DownloadedMovieImportService|Unknown Movie Adulthood
17-2-14 00:39:05.7|Debug|DiskScanService|Scanning '/mnt/ram/data_out/Radarr_drone/' for video files
17-2-14 00:39:05.7|Debug|DiskScanService|1 video files were found in /mnt/ram/data_out/Radarr_drone/
17-2-14 00:39:05.7|Debug|Parser|Parsing string 'Adulthood (2008) 720p'
17-2-14 00:39:05.7|Debug|Parser|Movie Parsed. Adulthood - 0 
17-2-14 00:39:05.7|Debug|Parser|Language parsed: English
17-2-14 00:39:05.7|Debug|QualityParser|Trying to parse quality for Adulthood (2008) 720p
17-2-14 00:39:05.7|Debug|Parser|Quality parsed: HDTV-720p v1
17-2-14 00:39:05.7|Debug|Parser|Release Group parsed: 
17-2-14 00:39:06.6|Debug|DownloadedMovieImportService|Unknown Movie for file: Adulthood (2008) 720p.mkv

This might be related to moving to Ubuntu, I had no issues when I was running on windows yet here I'm having issues with downloading disappearing from activity when its active in nzbget and seemingly doing nothing when completed.

**Comment by onedr0p** [comment]
Are you using drone folder and complete download handling?

**Comment by galli-leo** [comment]
Did you add the movie adulthood to your radarr library?

**Comment by shaggibruv** [comment]
I was only using the drone factory as a test, normally I was use the completed download handling.
I had added it to the library, scanned and it had downloaded it a few times but not picked it up for rename or even after I manually renamed it.

I think the problem might have been related to my own stupidity, I was mounting ACD with rclone as readwrite, since swapping to unionfs with read only and some HDD space radar seems to be working perfectly again with renaming.
